languages/elixir/exercises/concept/agent/.docs/after.md

```diff
@@ -0,0 +1,42 @@
+While spawning a process is easy, managing its state and behavior can become very complicated. The `Agent` module is an abstraction to facilitate this process to use an elixir process to manage a shared state.
```

Comment:
To prevent confusion, I would avoid here using the word "process" in its non-programming meaning. Maybe something like:
```suggestion
While spawning a process is easy, managing its state and behavior can become very complicated. The `Agent` module is an abstraction to facilitate managing a shared state in an Elixir process.
```